src/pages/playlists/viewPlaylist.js

```diff
@@ -5,11 +5,7 @@ import userService from "src/services/userService";
 import styled from "styled-components";
```

Comment:
In the application...after adding one playlist it is not possible to add another playlist

Reply:
I tested in preview, it seems to work fine with me

Reply:
ok...